### PR TITLE
[WIP] play with cnv periodic job

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml [hunk 1]
```diff
@@ -1236,3 +1236,82 @@ periodics:
...
+      - --secret-dir=/usr/local/e2e-aws-cluster-profile
+      - --target=e2e-aws
+      - --template=/usr/local/e2e-aws
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+        value: "registry.svc.ci.openshift.org/ocp/release:v4.1"
```
Comment (bparees) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml]: @rmohr fyi for now this appears to be working for rehearsal.  but for your actual job i think you can leave it empty.  if the rehearsal passes on this PR, i suggest you just update your PR to leave this field empty, we'll merge it, and see what happens when it runs for real.

File: ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml [hunk 2]
```diff
@@ -1236,3 +1236,82 @@ periodics:
...
+    job-test: e2e
+    job-release: '4.2'
+  name: canary-release-openshift-origin-installer-e2e-aws-4.2-cnv
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-aws-cnv-cluster-profile
```
Comment (bparees) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml]: @rmohr pretty sure this is why it was hanging.  the path here needs to align w/ your job name (e2e-aws-cnv)

File: ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml [hunk 3]
```diff
@@ -1236,3 +1236,82 @@ periodics:
...
+      - name: TEST_IMAGESTREAM_TAG
+        value: stable:hyperconverged-cluster-operator-tests
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/e2e-aws-cnv-cluster-profile
```
Comment (bparees) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml]: @rmohr also updated it here.

**Comment by rmohr** [comment]
@bparees thanks a lot for your help! Everything is passing now.

**Comment by bparees** [comment]
closing since this will go in via https://github.com/openshift/release/pull/4269

**Pull request closed without merging**